klog.go

```diff
@@ -1236,9 +1236,9 @@ func newVerbose(level Level, b bool) Verbose {
 // not evaluate its arguments.
 //
 // Whether an individual call to V generates a log record depends on the setting of
-// the -v and --vmodule flags; both are off by default. If the level in the call to
-// V is at least the value of -v, or of -vmodule for the source file containing the
-// call, the V call will log.
+// the -v and --vmodule flags; both are off by default. The V call with log if either
+// the value of -v is at least the level in the call to V, or if -vmodule matches the
+// source file containing the call.
```

Comment:
typo "with" / "will".

the statement is correct, yet this:

> -v is at least the level in the call to V

now reads as if -v depends on the level in V, which might confuse people as it's the other way around.

i'd phrase this like so:

> The V call will be logged if its level is less than or equal to the value of the -v flag, or alternatively if -vmodule matches the source file containing the call.

Reply:
Thanks for taking a look!

Updated in c95db6fa890f878eebae871f8afa94d50f1bb11b — actually, expanded a bit too, since `-vmodule` actually provides a mapping of module to level, so _just_ matching the source file isn't correct.